opi/plugins/ms_edge_beta.py

```diff
@@ -0,0 +1,22 @@
+import opi
+from opi.plugins import BasePlugin
+
+class MSEdgeBeta(BasePlugin):
+	main_query = "msedge-beta"
+	description = "Microsoft Edge Beta"
+	queries = ('microsoft-edge-beta', 'msedge-beta', 'edge-beta')
+
+	@classmethod
+	def run(cls, query):
+		if not opi.ask_yes_or_no("Do you want to install Microsoft Edge Beta from Microsoft repository?", 'y'):
+			return
+
+		opi.add_repo(
+			filename = 'microsoft-edge-beta',
+			name = 'Microsoft Edge Beta',
+			url = 'https://packages.microsoft.com/yumrepos/edge',
+			gpgkey = 'https://packages.microsoft.com/keys/microsoft.asc'
+		)
+
```

Comment:
If you look at the existing `ms_edge.py`, you will find that there is some trick needed to prevent the rpm post install script from messing with the repo configuration by creating a new repo: https://github.com/openSUSE/opi/blob/master/opi/plugins/ms_edge.py#L22

I guess that it is likely that we need a similar trick here as well.

You can view the script by running `rpm -qi --scripts microsoft-edge-beta`.
Please make sure that it is covered as well.
In the script look for something like `DEFAULTS_FILE="/etc/default/microsoft-edge-dev"`.
If that file exists and is empty, the post install script should do nothing.

Reply:
Ah - here it is:
```
DEFAULTS_FILE="/etc/default/microsoft-edge-beta"
```

So you will need this here:
```suggestion

		# tell rpm post script not to mess with our repos
		subprocess.call(['sudo', 'rm', '-f', '/etc/default/microsoft-edge-beta'])
		subprocess.call(['sudo', 'touch', '/etc/default/microsoft-edge-beta'])
```

Reply:
I'm on it.

Reply:
It's done!